File: news/src/main/java/com/leory/badminton/news/mvp/model/bean/MatchItemSection.java
package com.leory.badminton.news.mvp.model.bean;

import com.chad.library.adapter.base.entity.SectionEntity;

/**
 * Describe : 带有头部的bean
 * Author : leory
 * Date : 2019-05-20
 */
public class MatchItemSection extends SectionEntity<MatchItemBean> {
    public MatchItemSection(boolean isHeader, String header) {
        super(isHeader, header);
    }

    public MatchItemSection(MatchItemBean matchItemBean) {
        super(matchItemBean);
    }
}

File: news/src/main/java/com/leory/badminton/news/mvp/model/bean/MatchItemSection.kt
package com.leory.badminton.news.mvp.model.bean

import com.chad.library.adapter.base.entity.SectionEntity

/**
 * Describe : 带有头部的bean
 * Author : leory
 * Date : 2019-05-20
 */
class MatchItemSection : SectionEntity<MatchItemBean> {
    constructor(isHeader: Boolean, header: String) : super(isHeader, header) {}

    constructor(matchItemBean: MatchItemBean) : super(matchItemBean) {}
}
